### PR TITLE
Meditation_app_master

### PR DESCRIPTION
**Pull request by AlekseyGrimm**
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/stage-0/projects.md#task-3-meditation-app-20

### CODE REVIEW

File: Audio-App/app.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+replay.addEventListener("click", function() {
+    restartSong(song);
+    
+  });
+
+
+const restartSong = song =>{
+    let currentTime = song.currentTime;
+    song.currentTime = 0;
+    console.log("ciao")
```
Comment (y-solovey) [inline Audio-App/app.js]: please remove logs

File: Audio-App/app.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    video.play();
+    play.src = "./svg/pause.svg";
+  } else {
+    song.pause();
+    video.pause();
+    play.src = "./svg/play.svg";
+  }
+};
+
+song.ontimeupdate = function() {
```
Comment (y-solovey) [inline Audio-App/app.js]: please use adding listener `song.addEventListener('timeupdate', (event) => {
...
});`

File: Audio-App/app.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+outline.style.strokeDasharray = outlineLength;
+timeDisplay.textContent = `${Math.floor(fakeDuration / 60)}:${Math.floor(
+  fakeDuration % 60
+)}`;
+
+// playback of video and sounds
+sounds.forEach(sound => {
+  sound.addEventListener("click", function() {
+    song.src = this.getAttribute("data-sound");
+    video.src = this.getAttribute("data-video");
```
Comment (y-solovey) [inline Audio-App/app.js]: please use dataset

**Review by y-solovey — CHANGES_REQUESTED** [review]

File: Audio-App/app.js (hunk 2 above)
Comment (AlekseyGrimm) [reply Audio-App/app.js]: done

File: Audio-App/app.js (hunk 1 above)
Comment (AlekseyGrimm) [reply Audio-App/app.js]: done

**Review by y-solovey — APPROVED** [review]

File: Audio-App/app.js (hunk 3 above)
Comment (AlekseyGrimm) [reply Audio-App/app.js]: done

**Pull request merged**